frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/parent-or-guardian.tsx

```diff
@@ -20,9 +20,9 @@ import { RouteHandleData, getPathById } from '~/utils/route-utils';
 import { getTitleMetaTags } from '~/utils/seo-utils';
 
 export const handle = {
-  i18nNamespaces: getTypedI18nNamespaces('apply', 'gcweb'),
-  pageIdentifier: pageIds.public.apply.parentOrGuardian,
-  pageTitleI18nKey: 'apply:parent-or-guardian.page-title',
+  i18nNamespaces: getTypedI18nNamespaces('apply', 'apply-adult', 'gcweb'),
```

Comment:
```suggestion
  i18nNamespaces: getTypedI18nNamespaces('apply-adult-child', 'apply', 'gcweb'),
```

Reply:
does order matter if we prefix the namespace in the `t` function calls?

Reply:
no, but if you forgot to prefix it then it default to the first namespace in the list. The suggested order it's the pattern being use afaik.